erpnext/accounts/report/profitability_analysis/profitability_analysis.py

```diff
@@ -168,21 +168,24 @@ def get_columns(filters):
 			"label": _("Income"),
 			"fieldtype": "Currency",
 			"options": "currency",
-			"width": 120
+			"width": 305
```

Comment:
@nemesis189 Can you add some screenshots of the report in description. Want to see how it looks now. Also why 305 and 307 and not some rounded whole no like 320 or 300?

Reply:
adjusted the values for the table to fit the whole display, lesser values would leave a noticeable amount of space and looks quite asymmetric